generator-java/generators/app/templates/microservice/liberty/src/main/resources/index.html

```diff
@@ -0,0 +1,67 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+  <header id="flex-header">
+    <div class="cloud-header"></div>
+    <h1>Congratulations!</h1>
+    
+    <h2>You are currently running a WebSphere Liberty server built for the IBM Cloud.</h2>
+    </script>
```

Comment:
I found this rogue `</script>` end tag in the tool. It would be easiest for you to manually delete line 16 here.